app/javascript/channels/testRunChannel.ts

```diff
@@ -0,0 +1,29 @@
+import consumer from '../utils/action-cable-consumer'
+import { camelizeKeys } from 'humps'
+import { Submission } from '../components/student/Editor'
+import { TestRun } from '../components/student/editor/TestRunSummary'
+import { typecheck } from '../utils/typecheck'
+
+export class TestRunChannel {
+  subscription: ActionCable.Channel
+
+  constructor(submission: Submission, received: (testRun: TestRun) => void) {
+    this.subscription = consumer.subscriptions.create(
+      {
+        channel: 'Submission::TestRunsChannel',
```

Comment:
Can we change this to the real channel and remove this stubbed channel pls?

Reply:
This is not the stubbed channel. You might've been confused :)

Reply:
Probably by the word "Test" :)

Reply:
lol. Sorry. Thanks!